tests/e2e-instrumentation/instrumentation-java-multicontainer/00-install-collector.yaml

```diff
@@ -20,3 +20,6 @@ spec:
         traces:
           receivers: [otlp]
           exporters: [debug]
+        metrics:
```

Comment:
java auto inst can send metrics too, why not accept them :)

Reply:
+1